pages/pipelines/build_export.md

```diff
@@ -1,12 +1,35 @@
 # Export build data
 
 Each [Buildkite plan](https://buildkite.com/pricing) has a maximum build retention
-period, after which your oldest builds are automatically deleted.
+period. Once your builds reach the retention period, they will be removed from Buildkite.
 
 > 📘 Enterprise feature
 > Build export is only available on an [Enterprise](https://buildkite.com/pricing) plan.
 
-If you need to retain build data beyond the retention period in your [Buildkite plan](https://buildkite.com/pricing), you can have Buildkite export the data to a private Amazon S3 bucket. Prior to old build data being removed, Buildkite exports JSON representations of the builds to the S3 bucket you provide.
+If you need to retain build data beyond the retention period in your [Buildkite plan](https://buildkite.com/pricing), you can have Buildkite export the data to a private Amazon S3 bucket.
+
+Build export enables you to provide your own S3 bucket to export your build data to. If you do not provide an S3 bucket, Buildkite will export and store your builds into the Buildkite S3 Bucket (which can be requested at any time). Buildkite will store your exported build data for 12 months before deleting.
+
+## How it works
+Builds older than the build rentention limit will be automatically exported (in JSON format) to either the S3 bucket you have provided, or Buildkite's S3 bucket.
+
+You are able to configure you own S3 bucket in the UI. See [Enabling build exports](#enabling-build-exports) below.
+
+Buildkite stores your build data as multiple JSON files which include the following data:
+```
+buildkite/build-exports/org={UUID}/date={YYYY-MM-DD}/pipeline={UUID}/build={UUID}/
+    annotations.json.gz
+    artifacts.json.gz
+    build.json.gz
+    step-uploads.json.gz
+    jobs/
+        job-{UUID}.json.gz
+        job-{UUID}.log
+```
+
+The files will be stored in the following formats: [Annotations](https://buildkite.com/docs/apis/rest-api/annotations#list-annotations-for-a-build), [Artifacts](https://buildkite.com/docs/apis/rest-api/artifacts#list-artifacts-for-a-build), [Builds](https://buildkite.com/docs/apis/rest-api/builds#get-a-build), [Jobs](https://buildkite.com/docs/apis/rest-api/jobs#retry-a-job).
```

Comment:
`/docs/apis/rest-api/jobs#retry-a-job` shouldn't link to the “Retry a job” section…

But it's kinda weird that there's no “Get a build” section in the Jobs REST API docs. I guess that's because there's no API for that! API::JobsController#show doesn't exist… jobs are only shown embedded in Builds… which also seems weird to me, but that's a whole other thing.

Maybe for Jobs we should say “as would be embedded in a [Build via the REST API](https://buildkite.com/docs/apis/rest-api/builds#get-a-build)” rather than linking to the Jobs API.

Oh and maybe for Build we should say “but without `jobs`, which are stored in separate files.”

Reply:
Sweet. added your suggestion :)